### PR TITLE
Programing languages with letter P added in A-Z-dropdown.html file.

### PR DESCRIPTION
**Pull request by si-ddhant**
kindly give hacktoberfest-accepted tag. Thanks!!!

### CODE REVIEW

**Review by chilarai — COMMENTED** [review]
Why did you delete this?

**Comment by si-ddhant** [comment]
> Why did you delete alphabet `S`?

Alphabet S is still there i guess

File: A-Z-dropdown.html [hunk 1]
```diff
@@ -338,23 +338,6 @@
 <!-- End J -->
 
 <!-- Begin K -->
-<select name="select" id="select_K">
```
Comment (chilarai) [inline A-Z-dropdown.html]: My Bad. @si-ddhant, I am talking about this (alphabet `K` and not `S`). Why did you delete that?

Comment (si-ddhant) [reply A-Z-dropdown.html]: I was assigned with both the issues. Of K and P so i thought every issue requires different PRs. So i deleted it from this PR and did a new PR. Should I add the code for letter K too in this same PR?

Comment (chilarai) [reply A-Z-dropdown.html]: Please try to keep it clean and don't make unnecessary commits. Else it might go against Hactoberfest rules. You may do it separately but keep it clean

Here is the rule which says this

image

Comment (si-ddhant) [reply A-Z-dropdown.html]: Yeah my bad sorry .Should i commit the code for K in this same PR now?

Comment (chilarai) [reply A-Z-dropdown.html]: Yes. That will be better.

Comment (si-ddhant) [reply A-Z-dropdown.html]: it's added back now
